pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix

```diff
@@ -15,10 +15,6 @@ lib.packagesFromDirectoryRecursive {
     inherit (pkgs) codeium;
   };
 
-  consult-gh = callPackage ./manual-packages/consult-gh {
```

Comment:
One minor breaking change is that `consult-gh` from MELPA is just a subset of our `manualPackages.consult-gh`: `consult-gh-forge.el` and `consult-gh-embark.el` are not included in the MELPA version.  `consult-gh-embark.el` is to be added as separated package to MELPA in https://github.com/melpa/melpa/pull/9147.

Reply:
So, let's wait them?

Reply:
Yes, let's wait until the package has been fully migrated.

Reply:
Even when all three related packages are in MELPA, it is still a breaking change to replace one package with three packages.  But I am afraid nothing can be done to improve that in Nixpkgs.

In the future, to reduce breaking changes like this, should we suggest doing similar package separation for new manual packages?  Here is the MELPA [guide][1].  I do not find similar guide in the doc of GNU ELPA and NonGNU ELPA though.

[1]: https://github.com/melpa/melpa/blob/b2e8d53d3f3fbd628cb84347fe599ba531776ac1/CONTRIBUTING.org?plain=1#L160

Reply:
> In the future, to reduce breaking changes like this, should we suggest doing similar package separation for new manual packages? 

I think it is reasonable.
For all the intents and purposes, MELPA is defining what is a package, and asserting that a same source tree can generate various packages.
This is not an alien concept for us in Nixpkgs - e.g. `haskell-packages.Agda` provides two packages, Agda itself and agda2-mode.

The drawback is that it will require more caution and diligence from the package maintainer.
But I think it is part of the job.

Reply:
https://github.com/NixOS/nixpkgs/pull/348670 replaces the manually packaged `consult-gh` with the MELPA one.